smartrouter/image.yaml

```diff
@@ -0,0 +1,27 @@
+name: "jboss-bpmsuite-7/bpmsuite70-smartrouter"
+description: "Red Hat JBoss BPM Suite 7.0 Smart Router container image"
+version: "7.0.0"
+from: "ce-registry.usersys.redhat.com/jboss/openjdk18-rhel7:latest"
```

Comment:
Wrong image. Do not add registry part - it's internal URL. Version tag is wrong too.